source/qb64pe.bas

```diff
@@ -12574,19 +12574,30 @@ IF GenerateLicenseFile AND NOT NoExeSaved THEN
     makeline$ = makeline$ + " GENERATE_LICENSE=y"
 END IF
 
+'Clear nm output from previous runs
+FOR x = 1 TO ResolveStaticFunctions
+    IF LEN(ResolveStaticFunction_File(x)) THEN
+        s$ = tmpdir$ + "nm_output_" + StrReplace$(StrReplace$(ResolveStaticFunction_File(x), pathsep$, "."), ":", ".") + ".txt"
```

Comment:
It would be nice to make the filename conversion here into a separate function, then the logic wouldn't need to be repeated a few times down below. You could probably put it in `utilities/build.bas` as `CreateNMOutputFile$()` or something like that.